lib/models/debugs_bunny/concerns/has_guid.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+require 'active_support/all'
+
+module DebugsBunny
+  module HasGuid
+    extend ActiveSupport::Concern
+
+    REGEXP = /^[\w]+_[\w]+$/.freeze
```

Comment:
Don't think you need the character class `[]` for `\w`, since `\w` is already a character class. Also, I don't see any references to `REGEXP` in this PR. Is it referenced elsewhere?

Reply:
I got this file directly from our other repos that have a HasGuid concern; maybe this regex isn't necessary for this use case, so I will just remove it